docs/client-specifications.rst

```diff
@@ -108,7 +108,7 @@ Returns the list of collections and their current timestamp.
 
 .. note::
 
-    The ``_expected={}`` querystring parameter is mandatory but can be set to ``0`` if unknown. See next section about cache busting.
+    The ``_expected={}`` querystring parameter is mandatory. Either you receive a Push notification from the server, and pass the timestamp value in order to bust the CDN cache, or you use a hard-coded value (eg. ``0``) and rely on the cache TTL. See section below about cache busting.
```

Comment:
This one makes total sense to me.  Is this param is intended more for responding to push notifications than for polling?